src/components/join-components/joincontent.js

```diff
@@ -167,7 +203,7 @@ var innerContent2=[
       utilize our resources to provide their peers a platform to be heard and to 
       advocate for their community.
     </p>
-    <Row>
+    {/* <Row>
```

Comment:
We can start adding campus directors for this year. Let me know if you have trouble finding their info.